MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSHandledErrorLog.h

```diff
@@ -0,0 +1,21 @@
+#import <Foundation/Foundation.h>
+#import "MSAbstractErrorLog.h"
+
+@class MSException;
+
+/*
```

Comment:
`/**`

Reply:
Fixing, seems inconsistent in other places of the SDK as I copy pasted.